.gitignore

```diff
@@ -16,7 +16,7 @@ docs/book/
 packages/fuels/tests/**/**/Forc.lock
 
 # Don't add out/ files from test Sway projects.
-packages/fuels/tests/**/**/out/
+# packages/fuels/tests/**/**/out/
```

Comment:
IMO you need to remove this.

Reply:
This was intentional. 

> sway tests were compiled using forc from this PR: https://github.com/FuelLabs/sway/pull/3181

We can't get these sway files to work with the new VM without this version of forc. However it can't be released until the SDK is published with fuel-core v0.13 support. 

See this thread for more context: https://fuellabs.slack.com/archives/C02RU9MNVKN/p1666934602020589